e2e/jest.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   transform: {
     '\\.ts$': '<rootDir>/dist/index.js',
   },
-  testMatch: ['<rootDir>/e2e/__tests__/**/?(*.)+(spec|test).ts?(x)'],
+  testMatch: ['<rootDir>/e2e/__tests__/**/*.test.ts'],
```

Comment:
@kulshekhar There are some problems with `testMatch` on Windows and v24.1. It is reported here: [facebook/jest#7914](https://github.com/facebook/jest/issues/7914)

With actual code what surely works is: `testMatch: ['**/__tests__/**/*.test.ts'],` but it is a wider selector.

Reply:
my last push tried just that and it worked. Appveyor is still failing but only after running the e2e tests. It's the same issue I linked to a couple of days ago